packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md

```diff
@@ -52,7 +49,7 @@ A Generally Available (GA) feature is new or enhanced functionality that is enab
 
 Early Access (EA) features are opt-in features that you can try out in your org by asking Okta Support to enable them. Additionally, the **Features** page in the Okta Admin Console (**Settings** > **Features**) allows Super Admins to enable and disable some EA features themselves (assuming your org is eligible for the feature). 
 
-You can disable the EA features that you've enabled by deselecting them on the **Features** page. EA features that you disable are re-enabled by Okta automatically when the feature becomes GA.
+You can disable the EA features that you've enabled by clearing the associated boxes on the **Features** page. EA features that you disable are re-enabled by Okta automatically when the feature becomes GA.
```

Comment:
...you've enabled by clearing the associated *check boxes* on the **Features** page.

[According to the MS style guide - "Use check box, not box or checkbox, to refer to a check box in UI."]